tests/unit/amo/components/TestCollection.js

```diff
@@ -590,7 +593,22 @@ describe(__filename, () => {
     editLink.simulate('click', createFakeEvent());
 
     sinon.assert.calledWith(
-      dispatchSpy, openFormOverlay(COLLECTION_OVERLAY)
+      dispatchSpy, openFormOverlay(EDIT_COLLECTION_OVERLAY)
+    );
+  });
+
+  it('can open the Collection manager for creating', () => {
+    // Turn on the edit-overlay feature.
+    const fakeConfig = getFakeConfig({ enableCollectionEdit: true });
```

Comment:
I realized I don't need much of this, and I also added a test case for the `NotAuthorized` path, but I'm not going to push those changes yet, as I don't want them to hide all the comments I already added.